tests/ui/suggestions/dont_suggest_raw_pointer_syntax-issue-127562.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0594]: cannot assign to `*ptr`, which is behind a `*const` pointer
+  --> $DIR/dont_suggest_raw_pointer_syntax-issue-127562.rs:5:9
+   |
+LL |         *ptr = 3;
+   |         ^^^^^^^^ `ptr` is a `*const` pointer, so the data it refers to cannot be written
+   |
+help: consider specifying this binding's type
+   |
+LL |     let ptr: *mut i32 = std::ptr::addr_of!(val);
```

Comment:
This suggestion leads to ill-typed code so something is clearly still going wrong here.